test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildADesktopLibrary.cs

```diff
@@ -91,6 +92,113 @@ public void Method1()
 
         }
 
+        [WindowsOnlyFact]
+        public void It_can_reference_a_netstandard2_library_and_exchange_types()
+        {
+
+            var netStandardLibrary = new TestProject()
+            {
+                Name = "NETStandardLibrary",
+                TargetFrameworks = "netstandard2.0",
+                IsSdkProject = true
+            };
+
+            netStandardLibrary.SourceFiles["NETStandard.cs"] = @"
+public class NETStandard
+{
+    public static string GetString()
+    {
+        return ""Hello from netstandard2.0 library."";
+    }
+}
+";
+            var netFrameworkLibrary = new TestProject()
+            {
+                Name = "NETFrameworkLibrary",
+                TargetFrameworks = "net461",
+                IsSdkProject = true
+            };
+            netFrameworkLibrary.ReferencedProjects.Add(netStandardLibrary);
+
+            netFrameworkLibrary.SourceFiles["NETFramework.cs"] = @"
+public class NETFramework
+{
+    public void Method1()
+    {
+        string result = NETStandard.GetString();
+    }
+}
+";
+            var testAsset = _testAssetsManager.CreateTestProject(netFrameworkLibrary, "ExchangeNETStandard2")
+                .Restore(Log, netFrameworkLibrary.Name);
+
+            var buildCommand = new BuildCommand(Log, Path.Combine(testAsset.TestRoot, netFrameworkLibrary.Name));
+
+            buildCommand
+                .Execute()
+                .Should()
+                .Pass();
+        }
+
+        [WindowsOnlyFact]
+        public void It_can_use_ValueTuple_and_exchange_the_type_with_a_NETStandard_library()
+        {
+            var referenceAssemblies = ToolLocationHelper.GetPathToDotNetFrameworkReferenceAssemblies(TargetDotNetFrameworkVersion.Version47);
```

Comment:
I downloaded and installed it.  It's an optional component starting with 15.1.  Do you think we can install it on the CI machines?  Should be as simple as running https://www.microsoft.com/en-us/download/details.aspx?id=55168.